src/js/board.js

```diff
@@ -55,6 +55,7 @@ function createIdString(row, col) {
 function movePiece(piece, newRow, newCol) {
     const { row: oldRow, col: oldCol } = piece.getPosition();
     trackMoves.history.push({
+        pieceSymbol: piece.symbol, 
```

Comment:
Lets rename the object/key from "pieceSymbol" to "symbol" to keep it simple and consistent. :)